src/models/Interfaces/PortOption.ts

```diff
@@ -0,0 +1,8 @@
+export interface PortOption {
+    baudRate: number | undefined;

```

Comment:
Use `baudRate?: number` for optional member

Reply:
Fixed.